NEWS.md

```diff
@@ -1,3 +1,6 @@
+## 4.3.0 2018-11-27
```

Comment:
increment version, this one is already used in https://github.com/folio-org/mod-users-bl/pull/51/files

Reply:
Done. Added news for new version.